app/assets/stylesheets/common/base/_topic-list.scss

```diff
@@ -426,3 +426,22 @@ div.education {
   align-self: center;
   line-height: $line-height-medium;
 }
+
+.welcome-topic-banner {
+  border: 1px solid var(--primary-low);
+  box-shadow: 0px 0px 6px 0px rgba(0, 0, 0, 0.12);
+  padding: 10px;
+  display: flex;
+  flex-direction: row;
+
+  .col-text {
+    width: 70%;
```

Comment:
I suspect this won't look great on mobile?

Reply:
Curretly the banner only shows up on desktop view. For mobile view we'll need a new mockup as the existing layout does not looks good on mobile view and takes up lots of space.

Reply:
Ok, if possible then, we should move the relevant CSS to a desktop-only stylesheet.

Reply:
Sure, I've mode the CSS to desktop-only stylesheet.